reanalysis/interpretation_runner.py

```diff
@@ -133,7 +134,7 @@ def annotate_vcf(
     batch: hb.Batch,
     vep_temp: str,
     vep_out: str,
-    seq_type: SequencingType | None = SequencingType.GENOME,
+    seq_type: SequencingType = SequencingType.GENOME,
```

Comment:
Now that we have more exome data, I think we should be careful defaulting to Genome and always request it.

Reply:
Once we start running on Exomes there will be other changes required (at the moment there's no way to supply exome or genome as an argument/config), so I'm happy to assume this now and make an issue for 'things required for exomes'